src/main/java/seedu/address/ui/cards/PersonCard.java

```diff
@@ -0,0 +1,178 @@
+package seedu.address.ui.cards;
+
+import static seedu.address.ui.textstyle.TitleDescription.createTitleDescription;
+
+import java.util.Comparator;
+
+import javafx.fxml.FXML;
+import javafx.scene.control.Label;
+import javafx.scene.image.Image;
+import javafx.scene.image.ImageView;
+import javafx.scene.layout.FlowPane;
+import javafx.scene.layout.HBox;
+import javafx.scene.layout.Region;
+import javafx.scene.layout.VBox;
+import javafx.scene.text.Text;
+import javafx.scene.text.TextFlow;
+import seedu.address.model.person.Person;
+import seedu.address.ui.UiPart;
+import seedu.address.ui.textstyle.TitleDescription;
+
+/**
+ * An UI component that displays information of a {@code Person}.
```

Comment:
Really minor a UI not an UI

Reply:
The other JD comments are in the other PR, you can ignore them for now if you want

Reply:
Have fixed the java docs issues, thanks.